src/app/lib/config/services/urdu.js

```diff
@@ -185,6 +185,7 @@ export const service = {
     radioSchedule: {
       hasRadioSchedule: true,
       onFrontPage: false,
+      durationLabel: '%duration% دورانیہ',
```

Comment:
Awesome, so these have come back from the translation spreadsheet with the context provided! I'm slightly confused as you mentioned above you had to swap the order for arabic to be displayed correctly, is it different here?

Reply:
Here I used translation straight from the spreadsheet:
[hours, minutes, seconds]دورانیہ
For Arabic there was no translation in the spreadsheet so I just took it from Jon's comment here: https://github.com/bbc/simorgh/issues/5132#issuecomment-607250346
But I actually don't know if the label should go before or after the number

For Urdu I cannot even test this because `onFrontPage: false` despite the fact that `hasRadioSchedule: true`. So radio schedule is not even rendered for Urdu service.

Reply:
Took some extra time just to confirm, I think this was just a mistake. Correcting it now